tests/prague/eip7702_set_code_tx/test_set_code_txs.py

```diff
@@ -2943,6 +2943,28 @@ def test_contract_create(
     )
 
 
+def test_empty_authorization_list(
+    state_test: StateTestFiller,
+    pre: Alloc,
+):
+    """Test sending an invalid transaction with empty authorization list."""
+    tx = Transaction(
+        gas_limit=100_000,
+        to=pre.deploy_contract(code=b""),
+        value=0,
+        authorization_list=[],
```

Comment:
I vaguely remember @lightclient mentioning that this test would fail on the tx decoding part, which leads to failure of the state test. Tagging him to help us out on whether including this test would pose an issue.

Reply:
So this is ok then, or are you executing an invalid transaction?

Reply:
looks like a case for transaction tests

Reply:
Just checked on `evm` command and the state test and blockchain test both correctly reject the transaction.